### PR TITLE
Adds DisplayItem to items on DisplayWork

### PR DESCRIPTION
**Pull request by kenoir**
### What is this PR trying to achieve?

Preparing to add items to the Catalogue API post ingestion, in order to display items from the ontology to users. 

### Who is this change for?

💻  & 📚 

### Have the following been considered/are they needed?

- [ ] Deployed new versions

### CODE REVIEW

**Comment by kenoir** [comment]
For https://github.com/wellcometrust/platform-api/issues/572

File: api/src/main/scala/uk/ac/wellcome/platform/api/models/DisplayItem.scala [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  ) identifiers: Option[List[DisplayIdentifier]] = None,
+
+  @ApiModelProperty(
+    dataType = "List[uk.ac.wellcome.platform.api.models.DisplayLocation]",
+    value =
+      "List of locations that provide access to the item"
+  ) locations: Option[List[DisplayLocation]] = None,
+
+
+
```
Comment (alexwlchan) [inline api/src/main/scala/uk/ac/wellcome/platform/api/models/DisplayItem.scala]: What is all this whitespace?

Comment (kenoir) [inline api/src/main/scala/uk/ac/wellcome/platform/api/models/DisplayItem.scala]: a formatting error ...

**Comment by kenoir** [comment]
Depends https://github.com/wellcometrust/platform-api/pull/828

**Comment by kenoir** [comment]
For https://github.com/wellcometrust/platform-api/issues/572

**Pull request merged**